Engine/src/Application.bf

```diff
@@ -67,9 +151,114 @@ namespace SteelEngine
 			return true;
 		}
 
-		public void Dispose()
+
+		private void Update(float delta)
 		{
-			OnCleanup();
+			_inputManager.Update();
+
+			//DeleteQueuedComponents();
```

Comment:
Commented this out while trying to figure out the memory leaks and forgot to uncomment.